CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+Version 0.3.1 *(08/02/2017)*
```

Comment:
Will merging this trigger a build and release? If not, the release should go before updating this imo